scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -457,6 +501,10 @@ public void onClick(com.google.gwt.event.dom.client.ClickEvent event) {
 
         startAtRadioGroupPanel = new VerticalPanel();
         startAtRadioGroupPanel.setSpacing(10);
+
+        startAtRadioGroupPanel.add(startAccordingPlanningRB);
+        //  startAccordingPlanningRB.setVisible(false);
```

Comment:
Remove the commented code or explain why you keep it.

Reply:
yes !